wgsl/index.bs

```diff
@@ -6928,6 +6928,13 @@ That's not a full user-defined function declaration.
 
 ## Texture built-in functions ## {#texture-builtin-functions}
 
+In this section, texture types are shown with the following parameters:
+* |T|, a sampled type.
+* <var ignore>F</var>, a texel format.
```

Comment:
should these link to the relevant sections where texel formats and access modes are defined?

Reply:
Done in a followup.  Thanks for the feedback.